tests/base/src/test/java/org/wildfly/security/auth/client/MaskedPasswordSSLAuthenticationTest.java

```diff
@@ -85,7 +84,7 @@ public class MaskedPasswordSSLAuthenticationTest {
      * @return the initialised key manager.
      */
     private static X509ExtendedKeyManager getKeyManager(final String keystorePath) throws Exception {
-        KeyManagerFactory keyManagerFactory = KeyManagerFactory.getInstance(IS_IBM ? "IbmX509" : "SunX509");
```

Comment:
just a minor but does this work on other versions of IBM java? If not `KeyManagerFactory.getInstance(KeyManagerFactory.getDefaultAlgorithm())`  could be used instead

Reply:
Sorry I just noticed that SunX509 is accepted in [migration guide ](https://www.ibm.com/support/pages/node/6411285) to IBM java 11, so LGTM